### ISSUE TITLE
[Forge Mod] Forestry bees not showing in inventory

### ISSUE THREAD

**Issue by jmaccini** [jrbudda/minecrift#130]
![bees](https://cloud.githubusercontent.com/assets/1609940/16359430/651b7dda-3afa-11e6-98bd-607d0e6972d2.png)
Picture shows it best, but I have selected a bee in my inventory and it shows in my hand, but not in the inventory.  Doesn't show up in the apiaries or in chests either.  Honeycombs show up fine.

**Comment by jmaccini** [comment]
On v22.

**Comment by Techjar** [comment]
I sure wish I understood what's going on here. I looked into the item render code and I have no idea.

**Comment by jrbudda** [comment]
are those sections of the bee totally black?

**Comment by jmaccini** [comment]
Which sections?  This was happening with every type of bees I had in my apiaries.

This is what it would normally look like in an inventory:
![](http://ftbwiki.org/images/2/25/Grid_Majestic_Bee.png)

So there are black sections on the bees, but the background isn't typically blank.  It renders in my hand fine (And looks pretty cool!) It's just not showing in any inventories.

**Comment by jmaccini** [comment]
Hmm. I guess the grey border didn't copy over from [here](http://ftbwiki.org/Majestic_Bee).  Here's a screenshot:
![image](https://cloud.githubusercontent.com/assets/1609940/16419055/eca715de-3d11-11e6-87cf-1ed7d211ad6a.png)

**Comment by jrbudda** [comment]
I'm thinking the transparent part in your hand is due to those sections being solid black... I'm not sure about the inventory slot thing.

**Comment by jmaccini** [comment]
Oh, yeah that makes sense.  I wasn't too worried about that as the colors around the sides and the stripes on the bees are more important for telling them apart.  I actually didn't even notice the head wasn't filled in!

**Comment by GusSmedstad** [comment]
I’m pretty sure it’s related to how Forestry renders bees.

In Gregtech 5 unofficial, a fair number of things - mostly ores and liquids - share a single grayscale icon. The mod then tints the icon based on the material. So copper and iron use the same icon, but the mod tints one orange and the other white.

Forestry has a huge number of bees distinguished only by color. I’m guessing they’re doing the same thing.

Railcraft’s locomotives are also invisible in your inventory in Vivecraft. Railcraft’s locomotives have two colors, which the player can re-assign by dying the locomotive. Presumably there’s only one icon.

In a related note, Railcraft locomotives show up as gray when placed in the world, rather than the locomotive’s assigned colors.

Oddly enough, Gregtech’s stuff all renders correctly in Vivecraft, even though Forestry and Railcraft don’t. So obviously the details of the implementation are different.

**Comment by Techjar** [comment]
Yes, it's definitely some kind of blending issue, but we couldn't track it down. Kind of stopped caring cause of how old 1.7.10 is now, and the issue just went away with 1.10+ since that was a complete rewrite of the Forge integration.

**Comment by GusSmedstad** [comment]
Unfortunately, there are a number of mods I’m using that never got updated past 1.7.10. Gregtech 5U, Nuclear Control, and Magneticraft in particular.
